styles.css

```diff
@@ -1,26 +1,16 @@
-/* font-family: 'Quicksand', sans-serif;
-COLORS
-light-yellow - #f7e4bf;
-light-blue - #78a7c6;
-dark-blue - #134d71;
-white - #ffffff;
-black - #000000;
-*/
 
```

Comment:
Delete these empty lines

Reply:
Lines deleted, there should be no existing commented out sections except for the ones that describe breaking points or provide dev empathy.